atlas/map_test.go

```diff
@@ -345,7 +345,7 @@ func TestEncode(t *testing.T) {
 					},
 				},
 			},
-			tile: slippy.NewTile(2, 3, 4),
+			tile: slippy.NewTile(2, 3, 3),
```

Comment:
4 is not a valid y-value at zoom 2, and the geom upgrade makes this test fail unless a valid coordinate is used.